rita/src/middleware.rs

```diff
@@ -0,0 +1,36 @@
+use actix_web::middleware::{Middleware, Response, Started};
+use actix_web::{HttpRequest, HttpResponse, Result};
+use http::{header, HttpTryFrom, Method, StatusCode};
+use regex::Regex;
+
+pub struct Headers; // <- Our middleware
+
+/// Middleware implementation, middlewares are generic over application state,
+/// so you can access state with `HttpRequest::state()` method.
```

Comment:
Are these comments from copy-pasted example code? IMO comments are only useful when you are doing something surprising or very unusual, not annotating obvious code.

Reply:
Yeah, think these comments should be removed